tests/model_fields/test_decimalfield.py

```diff
@@ -91,7 +91,10 @@ def test_lookup_really_big_value(self):
         Really big values can be used in a filter statement.
         """
         # This should not crash.
-        Foo.objects.filter(d__gte=100000000000)
+        self.assertQuerySetEqual(Foo.objects.filter(d__gte=100000000000), [])
```

Comment:
For some context, the test was introduced in e9814029f570bd0866dc859147bca90340bcc913 to address ticket-10692 but since the queryset wasn't evaluated I think it was broken from the beginning.

Unless it was changed during the expressions refector, which I doubt in this case, `get_db_prep_value` and `get_db_prep_save` method are only called at queryset compilation time (see `Lookup.as_sql`) and not at resolving time (`QuerySet.filter`).

Reply:
Evaluating the queryset forces compilation so I think this is a good enough test.